test/unit/lib/meli-client.test.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/first,import/order,import/no-unassigned-import */
+import test from 'ava'
+
+// Load test env variables
+const envTestPath = require('path').resolve(process.cwd(), '.env.test')
+require('dotenv').config({path: envTestPath})
+
+const MeliClient = require('../../../lib/meli-client')
+const Account = require('../../../model/account')
+
+// Mount db connection
+require('../../../config/db')
```

Comment:
is not cool to make stuff when you require something, is better if you require a function `init()` or something like `require('../../../config/db')()` or `require('../../../config/db').config()` so the developer user of this tools now that he is actually doing something with this.

Reply:
good suggestion (Y) I'll apply it